test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -4,6 +4,14 @@
 
```

Comment:
this is going to have some rough conflicts w #883

Reply:
The merge conflicts will look horrible for sure, but looking at that PR they'll be pretty easy to resolve manually if I rebase on top of that after merge - I really only have to add a couple fields here and there.